src/channel.coffee

```diff
@@ -86,6 +86,19 @@ class Channel
     m = new Message @_client, {text: text}
     @sendMessage m
 
+  postMessage: (data) ->
+    params = data
+    params.channel = @id
+    if data.attachments
+      params.attachments = JSON.stringify(data.attachments)
+
+    @_client.logger.debug data
+    @_client.logger.debug params
+    @_client._apiCall "chat.postMessage", params, @_onPostMessage
+
+  _onPostMessage: (data) =>
```

Comment:
Just a thought...

There are a bunch of these `_on{{ApiMethod}}` methods that just call the logger.  Not really *bad* but might be easier to just pass around the logger directly instead of a bunch of different methods that wrap it (unless there's some other purpose I'm missing).